db/schema.rb

```diff
@@ -10,11 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_04_09_175743) do
+ActiveRecord::Schema.define(version: 2021_04_12_104001) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "calculators", force: :cascade do |t|
+    t.uuid "uuid", null: false
+    t.string "name"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["uuid"], name: "index_calculators_on_uuid", unique: true
+  end
+
```

Comment:
update your branch. This is already in master

Reply:
Thanks, done.